### ISSUE TITLE
Creating your first app with VS Code topic

### ISSUE THREAD

**Issue by gregvanl** [Microsoft/vscode-docs#126]
Update the Node.js topic to start with a simple hello world example.  We can then dive into doing something useful with express, etc.  
We should link to this section from our the Basics and overview topics so users can get something running quickly with VS code and little other setup required.

**Comment by gregvanl** [comment]
Turning this into a hero story.  First time user can get a simple application running in VS Code in minutes.

**Comment by JanKok** [comment]
As a newcomer to VS Code and one who has never even seen Node.js code before, I strongly agree! Need a very short and easy (since my objective is to develop my C program, not to learn Node.js) introduction for those who aren't familiar with Node.js.

On the https://code.visualstudio.com/docs/runtimes/nodejs page I got stuck at this line: "To get started, install Node.js for your platform."

That link (https://nodejs.org/download/) took me to a page which in Firefox on my Ubuntu system looks like:

Index of /download/

../
docs/                                              23-Feb-2016 05:31                   -
next-nightly/                                      30-Aug-2015 08:29                   -
nightly/                                           26-Feb-2016 00:05                   -
rc/                                                26-Feb-2016 10:50                   -
release/                                           23-Feb-2016 05:31                   -
test/                                              23-Feb-2016 07:00                   -

As I newcomer I have no idea what I'm supposed to do to install Node.js, or even if it's necessary (since VS Code already uses Node.js, is it even necessary to install anything else?)

I spent about two hours just trying to figure that out, then gave up and skipped the rest of the nodejs page as well as the page above it, https://code.visualstudio.com/docs/editor/debugging .

I hope someone can improve this part of the documentation soon. Then I can review it with my newcomer hat still on and give more feedback if things still aren't clear.
